docs/Settings.md

```diff
@@ -169,7 +169,8 @@ These parameters are accessed by calling [GetParam](./api/GetParam.md) or [SetPa
 | `QUIC_PARAM_CONN_STATISTICS_V2`<br> 22            | QUIC_STATISTICS_V2            | Get-only  | Connection-level statistics, version 2.                                                   |
 | `QUIC_PARAM_CONN_STATISTICS_V2_PLAT`<br> 23       | QUIC_STATISTICS_V2            | Get-only  | Connection-level statistics with platform-specific time format, version 2.                |
 | `QUIC_PARAM_CONN_ORIG_DEST_CID` <br> 24           | uint8_t[]                     | Get-only  | The original destination connection ID used by the client to connect to the server.       |
-
+| `QUIC_PARAM_CONN_THIS_RELIABLE_RESET_ENABLED` 25  | uint8_t (BOOLEAN)             | Both      | Indicate/query support for QUIC CLOSE_STREAM frames. Must be set before start.            |
+| `QUIC_PARAM_CONN_PEER_RELIABLE_RESET_ENABLED` 26  | uint8_t (BOOLEAN)             | Get-only  | Indicates peer advertised support for QUIC CLOSE_STREAM frames.                           |
```

Comment:
Let's only have 1 option. Do you enable this feature or not?

Reply:
Do we not care about the peer?

Reply:
You can start with it enabled. After the handshake it reflects the negotiated value.

Reply:
Ahh I see, so start the flag to be off. If the app turns it on, then connects to a server, if the server says their flag is turned off (passed through connection), then we turn ours off (overriding the app's decision)?

Reply:
Essentially.